pandas/core/frame.py

```diff
@@ -5032,7 +5032,7 @@ def swaplevel(self, i=-2, j=-1, axis=0):
 
         Returns
         -------
-        swapped : same type as caller (new object)
+        same type as caller (new object)
```

Comment:
Hmm I think when we've done this for other objects we've said something like `DataFrame or Series` (or whatever is applicable) and left `Same type as caller` as a comment

Reply:
I switched this to `DataFrame`, since that should always be the type of the caller here